pkg/storegateway/chunkscache/cache.go

```diff
@@ -0,0 +1,144 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+
+package chunkscache
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	gokit_log "github.com/go-kit/log"
```

Comment:
[nit] No need to rename it.

```suggestion
	"github.com/go-kit/log"
```